libs/utils/android/workloads/youtube.py

```diff
@@ -66,9 +66,10 @@ def run(self, out_dir, video_url, video_duration_s, collect=''):
         self.collect = collect
 
         # Unlock device screen (assume no password required)
-        System.menu(self._target)
-        # Press Back button to be sure we run the video from the start
-        System.back(self._target)
+        Screen.unlock(self._target)
+
+        # Stop youtube if already running
+        System.force_stop(self._target, self.package)
```

Comment:
Should not this better go before the unlock?

However, I would prefer to keep this chunk into a separate patch, which follow the one updating the unlock procedure. At the end that's required just by Youtube and to grant that we start from the beginning.